server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -582,6 +585,19 @@ export class SummaryWriter implements ISummaryWriter {
         pending: ISequencedOperationMessage[]): Promise<ISequencedDocumentMessage[]> {
         if (lt - gt <= 1) {
             return [];
+        } else if (pending.length > 0) {
+            // Check for logtail ops in the unprocessed ops currently present in memory
+            const pendingOps = pending.filter((op) => op.operation.sequenceNumber > gt && op.operation.sequenceNumber < lt).map((op) => op.operation);
+            if (pendingOps.length > 0 && pendingOps[0].sequenceNumber === gt + 1 && pendingOps[pendingOps.length - 1].sequenceNumber === lt - 1) {
```

Comment:
>pendingOps[pendingOps.length - 1] [](http://example.com/codeflow?start=84&length=33)

(nit) pendingOps.at(-1) for a couple of fewer characters? :smile:

Reply:
Unfortunately, this was introduced only in [ECMA script 2022](https://dev.to/jasmin/whats-new-in-es2022-1de6#:~:text=1.%20Method%20.at%28%29%20of%20indexable%20values.)

Reply:
Ah...didn't realize that